compair/api/__init__.py

```diff
@@ -4,14 +4,19 @@
 from flask import redirect, render_template, jsonify
 from flask_login import login_required, current_user, current_app
 from flask import make_response
-from flask import send_file, url_for
+from flask import send_file, url_for, redirect
 from flask_restful.reqparse import RequestParser
 
+from bouncer.constants import READ
+from compair.authorization import require
+from compair.kaltura import KalturaAPI
+from compair.models import File
 from compair.core import event
 on_get_file = event.signal('GET_FILE')
 
 attachment_download_parser = RequestParser()
 attachment_download_parser.add_argument('name', default=None)
+attachment_download_parser.add_argument('file_id', default=None)
```

Comment:
I don't see why you need this? (file.name is uuid with file extension so it should identify the file)

Reply:
maybe we should make file.name unique (indexed) so we can just query that.